mappings/net/minecraft/world/gen/feature/SwampHutFeature.mapping

```diff
@@ -1,7 +1,7 @@
 CLASS net/minecraft/class_3197 net/minecraft/world/gen/feature/SwampHutFeature
 	FIELD field_13882 MONSTER_SPAWNS Ljava/util/List;
 	FIELD field_16435 CREATURE_SPAWNS Ljava/util/List;
-	METHOD method_14029 (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;)Z
+	METHOD method_14029 isAHut (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;)Z
```

Comment:
```suggestion
	METHOD method_14029 isHut (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;)Z
```

Reply:
yeah I put that at first but then thought about it and thought that it is a single hut thanks for the input.

Reply:
we tend to dislike `a` or `the` in names unless it appears in registry name like `the_nether` `the_end`

Reply:
Changed and thanks for the explanation.
https://github.com/FabricMC/yarn/pull/1218/commits/be664dcb4efada12f110bad0510572357e871453